packages/itwinui-react/src/core/Stepper/Stepper.tsx

```diff
@@ -44,6 +45,30 @@ export type StepperProps = {
    *  Click handler on completed step.
    */
   onStepClick?: (clickedIndex: number) => void;
+  /**
+   * Callback that can provide additional props for `<li>` representing a step.
+   */
+  stepProps?: (index: number) => React.ComponentProps<'li'>;
+  /**
+   * Allows props to be passed for track-content
```

Comment:
I don't think the hyphen is needed for `track-content` and `label-count`

Reply:
fixed

Reply:
Nitpicky, but I think we use sentence notation for jsdocs, meaning each of these comments need punctuation at the end

Reply:
fixed